test/TestAll.cpp

```diff
@@ -0,0 +1,13 @@
+#include <gtest/gtest.h>
+#include "CollisionDetectorTest.cpp"
+#include "GameObjectTest.cpp"
+#include "GameStateHandlertest.cpp"
+#include "HitboxTest.cpp"
+#include "KeyboardListenerTest.cpp"
+#include "MissileLauncherTest.cpp"
```

Comment:
I think there's a call in CMakeLists to add tests. https://cmake.org/cmake/help/v3.0/module/FindGTest.html I would try that, including cpp files can make some really funky errors :/